pkgs/development/python-modules/prometheus-pandas/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipython
+, matplotlib
+, numpy
+, pandas
+, requests
+}:
+
+buildPythonPackage rec {
+    pname = "prometheus-pandas";
```

```suggestion
  pname = "prometheus-pandas";
```
We usually use two space indent.

---

Ah sorry. Fixed up.